### ISSUE TITLE
doc for user functions and variables

### ISSUE THREAD

**Issue by AmjadHD** [julia-vscode/LanguageServer.jl#268]
documentations only work for Base and Core variables, but not for user's
```julia
"""
    myfunc(x::AbstractString, y::AbstractString)

documentation that should be displayed.
"""
function myfunc(x::AbstractString, y::AbstractString)
    # Code Here
end
```
Is it not supported yet ?

**Comment by non-Jedi** [comment]
@AmjadHD, I'm seeing docs work fine for any package when using this package with eglot in emacs. Is this an issue you still have?

**Comment by AmjadHD** [comment]
yes
![image](https://user-images.githubusercontent.com/33235928/66437727-daafef80-ea2a-11e9-992c-0c9c89d81495.png)

**Comment by non-Jedi** [comment]
I can reproduce.

To clarify this issue: the problem is that symbols defined within the environment over which the language server is running don't show docstrings. Doc help is available for symbols outside of `Core`/`Base` but not for symbols defined within in the text being edited. So e.g. you can get documentation for `I` if there's a `using LinearAlgebra` in the file but not for `I2`:

```julia
using LinearAlgebra

"Identity matrix multiplied by 2"
I2() = 2I
```

**Comment by ZacLN** [comment]
Addressed by #438
